net/net-snmp/patches/750-ieee802dot11.patch

```diff
@@ -1,6 +1,5 @@
-diff -Naur a/agent/mibgroup/ieee802dot11.c b/agent/mibgroup/ieee802dot11.c
---- a/agent/mibgroup/ieee802dot11.c	1970-01-01 07:00:00.000000000 +0700
-+++ b/agent/mibgroup/ieee802dot11.c	2024-05-08 20:19:26.050637500 +0700
+--- /dev/null
```

Comment:
hmmm?

Reply:
What do you mean?, isn't it's there since the beginning?.

https://github.com/openwrt/packages/blob/master/net/net-snmp/patches/750-ieee802dot11.patch#L1